### PR TITLE
Update to support new OmniSharp release

### PR DESCRIPTION
**Pull request by DustinCampbell**
This change updates the C# extension to the latest v1.9-alpha10 release of OmniSharp. In addition, I have removed the DNX restore and execute commands, and updated the dotnet restore command.

cc @gregg-miskelly, @chuckries, @Pilchie

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,8 +45,6 @@
     "onLanguage:csharp",
     "onCommand:o.restart",
     "onCommand:o.pickProjectAndStart",
-    "onCommand:o.restore",
-    "onCommand:o.execute",
     "onCommand:o.showOutput",
     "onCommand:o.execute-last-command",
```
Comment (gregg-miskelly) [inline package.json]: Remove execute-last-command also?

Comment (DustinCampbell) [inline package.json]: Good catch -- thanks!

**Comment by gregg-miskelly** [comment]
:+1:

**Comment by Pilchie** [comment]
:+1:

**Pull request merged**